corda-common-flows/src/main/java/ch.cordalo.corda.common.flows/SimpleBaseFlow.java

```diff
@@ -74,6 +83,39 @@
         return signSyncCollectAndFinalize(state.getParticipants(), transactionBuilder);
     }
 
+    @Suspendable
```

Comment:
Name should be
SearchByLinearId

And generic method missing for any criteria

Reply:
If method extended to other criteria there should be a call back method to be implanted. So create new APi in interfaces of SimpleFlow